<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab">
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QLabel" name="seedLabel">
         <property name="text">
          <string>Seed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="itersLabel">
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="bailLabel">
         <property name="text">
          <string>Bailout</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="fillG">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="fillR">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="fill">
         <property name="text">
          <string>Fill</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QSpinBox" name="fillB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QSpinBox" name="seed">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QSpinBox" name="bail">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QSpinBox" name="iters">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="solid">
      <attribute name="title">
       <string>Solid</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="2">
        <widget class="QSpinBox" name="solidG">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="solidLabel">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="solidR">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="solidB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="periodic">
      <attribute name="title">
       <string>Periodic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="smooth">
         <property name="text">
          <string>Smooth</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="orbitrap">
      <attribute name="title">
       <string>Orbitrap</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Trap</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="trap">
         <item>
          <property name="text">
           <string>Center</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Axes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Circular</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="density">
      <attribute name="title">
       <string>Density</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="layersLabel">
         <property name="text">
          <string>Layers</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="samplesLabel">
         <property name="text">
          <string>Samples</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="samples">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="layers">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
